app/views/planning_applications/assessment/consultees/index.html.erb

```diff
@@ -0,0 +1,45 @@
+<% content_for :page_title do %>
+  Confirm all consultees have been consulted - <%= t("page_title") %>
+<% end %>
+
+<% add_parent_breadcrumb_link "Home", root_path %>
+<% add_parent_breadcrumb_link "Application", planning_application_path(@planning_application) %>
+<% add_parent_breadcrumb_link "Consultation", planning_application_consultation_path(@planning_application) %>
+<% content_for :title, "Confirm all consultees have been consulted" %>
```

Comment:
Sorry, just noticed this, we have a shared component that deals with this. 

e.g.

```
<%= render(
      partial: "shared/assessment_task_breadcrumbs",
      locals: { planning_application: @planning_application }
    ) %>

<% content_for :title, "Add #{"pre-commencement" if @condition_set.pre_commencement?} conditions" %>
```